pkg/apis/serving/v1beta1/component.go

```diff
@@ -45,8 +45,7 @@ const (
 	InvalidProtocol                                  = "invalid protocol %s. Must be one of [%s]"
 	MissingStorageURI                                = "the InferenceService %q is invalid: StorageURI must be set for multinode enabled"
 	InvalidAutoScalerError                           = "the InferenceService %q is invalid: Multinode only supports 'none' autoscaler(%s)"
-	InvalidNotSupportedStorageURIProtocolError       = "the InferenceService %q is invalid: Multinode only supports 'pvc' Storage Protocol(%s)"
-	InvalidCustomGPUTypesAnnotationFormatError       = "the InferenceService %q is invalid: invalid format for %s annotation: must be a valid JSON array"
```

Comment:
wonder why `InvalidCustomGPUTypesAnnotationFormatError` needs to be removed?

Reply:
Simple cleanup. The constant is unused.

Reply:
sounds good